docs/content/installation/install_hostdefinition.md

```diff
@@ -0,0 +1,36 @@
+# Installing the `HostDefinition` custom resource
+
+Install the `HostDefinition` custom resource to enable automatic host definitions from within the CSI driver.
+
+Use the following steps to install the `HostDefinition` custom resource, with [GitHub](https://github.com/IBM/ibm-block-csi-operator).
+
+**Note:** Only hosts added after installation are automatically defined with the CSI driver.
```

Comment:
IMO it's not clear that `hosts added` means added to the csi node pod deployment

Reply:
changed